I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 238, in script
    play sound3 door_creak
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
Exception: Audio channel 'sound3' is unknown.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 238, in script
    play sound3 door_creak
  File "/Users/happypuffin/Desktop/code/renpy-8.0.3-sdk/renpy/ast.py", line 2232, in execute
    self.call("execute")
  File "/Users/happypuffin/Desktop/code/renpy-8.0.3-sdk/renpy/ast.py", line 2220, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "/Users/happypuffin/Desktop/code/renpy-8.0.3-sdk/renpy/statements.py", line 281, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 123, in execute_play_music
    renpy.music.play(_audio_eval(p["file"]),
  File "/Users/happypuffin/Desktop/code/renpy-8.0.3-sdk/renpy/audio/music.py", line 103, in play
    c = get_channel(channel)
  File "/Users/happypuffin/Desktop/code/renpy-8.0.3-sdk/renpy/audio/audio.py", line 915, in get_channel
    raise Exception("Audio channel %r is unknown." % name)
Exception: Audio channel 'sound3' is unknown.

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.0.3.22090809
Горе От Ума 1.0
Sat Jan 28 21:28:24 2023
